src/main/kotlin/com/mirf/features/dicomimage/data/DataConvert.kt

```diff
@@ -0,0 +1,120 @@
+package com.mirf.features.dicomimage.data
+import com.pixelmed.dicom.TagFromName
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
```

Comment:
As far as I understand from your code, all of the functions in this class are used in DicomData as util functions. If so, you can just make them private in DicomData class. Otherwise, you should group them to be static method of some class (smth like DicomDataConverter). It is not a good practise to use random methods without class for OOP programming.